store/localstore/kv.go

```diff
@@ -150,9 +153,10 @@ func (s *dbStore) scheduler() {
 			}
 		case <-s.closeCh:
 			closed = true
-			s.wg.Done()
 			// notify seek worker to exit
 			close(seekCh)
+			wgSeekWorkers.Wait()
+			s.wg.Done()
```

Comment:
It should't, we still need to return error when store closed if user try to commit transaction.